src/quo2/foundations/colors.cljs

```diff
@@ -185,95 +186,130 @@
+                                30 "#B7E9D0"
+                                40 "#8FDCB7"
+                                50 "#5BCC95"
+                                60 "#4CAB7D"
+                                70 "#3E8B65"
+                                80 "#2F6A4E"
+                                90 "#214936"}
+                    :yellow    {10 "#FFFDF8"
+                                20 "#FFF5DD"
+                                30 "#FFE8B3"
+                                40 "#FFDC8A"
+                                50 "#FFCB53"
+                                60 "#D6AA46"
+                                70 "#AD8A38"
+                                80 "#856A2B"
+                                90 "#5C491E"}
+                    :orange    {10 "#FFFBF9"
+                                20 "#FEE9DF"
+                                30 "#FDCEBA"
+                                40 "#FCB394"
+                                50 "#FB8F61"
+                                60 "#D37851"
+                                70 "#AB6142"
+                                80 "#824A32"
+                                90 "#5A3323"}
+                    :red       {10 "#FFF9F9"
+                                20 "#FDE0E0"
+                                30 "#FABCBC"
+                                40 "#F89797"
+                                50 "#F46666"
+                                60 "#CD5656"
+                                70 "#A64545"
+                                80 "#7F3535"
+                                90 "#582525"}
+                    :pink      {10 "#FFFAFC"
+                                20 "#FEE5EE"
+                                30 "#FEC5DA"
+                                40 "#FDA5C6"
+                                50 "#FC7BAB"
+                                60 "#D46790"
+                                70 "#AB5474"
+                                80 "#834059"
+                                90 "#5B2C3E"}
+                    :brown     {10 "#FBF9F8"
+                                20 "#EBDFDB"
+                                30 "#D2B9B1"
+                                40 "#BA9386"
+                                50 "#99604D"
+                                60 "#805141"
+                                70 "#684134"
+                                80 "#503228"
+                                90 "#37231C"}
+                    :beige     {10 "#FDFCFB"
+                                20 "#F4EFE9"
+                                30 "#E8DBD0"
+                                40 "#DBC8B6"
+                                50 "#CAAE93"
+                                60 "#AA927C"
+                                70 "#897664"
+                                80 "#695A4C"
+                                90 "#493F35"}})
+(defn custom-color
+  ([color suffix]
+  (custom-color color suffix nil))
+  ([color suffix opacity]
+   (let [base-color get-in customisation [(keyword color) suffix]]
```

Comment:
how it works without `( )` ?

Reply:
oh sorry, I was playing around with different approach `@(resolve ...)` and in the end missed brackets while pushing code. will fix it. thank you